elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/DisableGraphQLAsyncTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example.tests;
+
+import static com.yahoo.elide.Elide.JSONAPI_CONTENT_TYPE;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attr;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attributes;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.data;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.id;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.resource;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.type;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.ActiveProfiles;
+
+/**
+ * Executes Async tests with Aggregation Store disabled.
+ */
+@ActiveProfiles("disableGraphQL")
```

Comment:
Rather than create an entire application.yaml for this one test, you can set individual settings for tests using code like:

```java
@TestPropertySource(
        properties = {
                "elide.json-api.enableLinks=true"
        }
)
```